testsuite/gnat2goto/tests/address_clause/test.out

```diff
@@ -18,6 +18,6 @@ N_Attribute_Definition_Clause "address" (Node_Id=2295) (source,analyzed)
  Entity = N_Defining_Identifier "var_addr_2" (Entity_Id=2282)
  Check_Address_Alignment = True
 
-[address_clause.assertion.2] line 12 assertion Var_Addr_1 + Var_Addr_2 = 3: SUCCESS
```

Comment:
I'm confused why this change is showing up, when the exact same change is made in #330: https://github.com/diffblue/gnat2goto/pull/330/files#diff-b450147fb1fc8092f271746d6b36256e but I suppose it doesn't matter

Reply:
I think it is simply that it is based on an old commit of master. If you rebase first, your commit will be a little smaller, but I don't mind either way

Reply:
Yeah, I think this should be rebased so that the PR is against the newest commit of master - though possibly in Spanners defence, I did have a PR recently (in a different Github repo) where even though I'd rebased my code, force pushed it to the PR, etc - the PR was *still* showing the old version. In that instance, I changed the commit message slightly and then did another force push, and that seemed to be enough to get the PR showing the right thing. So @spanners - if you could double check you've done a fetch/rebase, and try pushing again, it would be good to have this PR showing the proper diff.

Reply:
https://github.com/diffblue/gnat2goto/pull/327#discussion_r370011366 I *think* I've rebased now! Thanks for your patience.